modules/mono/editor/GodotTools/GodotTools/Export/ExportPlugin.cs

```diff
@@ -169,17 +177,44 @@ private void _ExportBeginImpl(string[] features, bool isDebug, string path, long
                         "Publish succeeded but project assembly not found in the output directory");
                 }
 
-                // Add to the exported project shared object list.
+                var manifest = new StringBuilder();
 
+                // Add to the exported project shared object list or packed resources.
                 foreach (string file in Directory.GetFiles(publishOutputTempDir, "*", SearchOption.AllDirectories))
                 {
-                    AddSharedObject(file, tags: null,
-                        Path.Join(projectDataDirName,
-                            Path.GetRelativePath(publishOutputTempDir, Path.GetDirectoryName(file))));
+                    if (embedBuildResults)
+                    {
+                        var filePath = SanitizeSlashes(Path.GetRelativePath(publishOutputTempDir, file));
```

Comment:
I'm curious why sanitizing the path is needed for android exports.

Reply:
Not specific to android, but if the pck/zip contains files using (some) backslashes as the path separator, these files cannot be retrieved, as during lookup it will hash the path using forward slashes, and then be different to the hash with backslashes.